servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcServerBuilder.java

```diff
@@ -63,6 +63,7 @@ public GrpcServerBuilder protocols(final HttpProtocolConfig... protocols) {
         return this;
     }
 
+    @Deprecated
     @Override
     public GrpcServerBuilder backlog(final int backlog) {
         httpServerBuilder.backlog(backlog);
```

Comment:
Why is this not using the approach suggested in the abstract class?

Reply:
I will use your suggestion above todo this in the base class. the base method will be removed in future releases.